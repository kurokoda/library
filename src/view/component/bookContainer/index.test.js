import React from "react";
import renderer from "react-test-renderer";

import BooksContainer from ".";

const props = {
  buttonLabel: "Button Label",
  label: "Label",
  onBookButtonClick: () => {},
  onPageChange: () => {},
};

it("renders correctly", () => {
  const tree = renderer.create(<BooksContainer {...props} />).toJSON();
  expect(tree).toMatchSnapshot();
});
