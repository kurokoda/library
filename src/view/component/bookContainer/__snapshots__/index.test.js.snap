// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="library__books-container"
>
  <h3>
    Label
  </h3>
  <div>
    No Data
  </div>
</div>
`;
